packages/react-native-bpk-component-calendar/src/BpkCalendar.ios.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  type StyleObj,
+  Button,
+  StyleSheet,
+  View,
+  ViewPropTypes,
+  requireNativeComponent,
+  NativeModules,
+} from 'react-native';
+
+const RCTCalendarView = requireNativeComponent('RCTCalendarView');
+
+const styles = StyleSheet.create({
+  base: {
+    width: '100%',
+    height: '100%',
+  },
+});
+
+export type Props = {
+  style: ?StyleObj,
+};
+
+const getDateInOneYear = () => {
+  const date = new Date();
+  const newMonth = date.getMonth() + 12;
+  date.setMonth(newMonth);
+  return date;
+};
```

Comment:
The naming of and within this function feels a bit generic, I think it could be more descriptive. What about something like:

```
const getDateOneYearFromNow = () => {
  const today = new Date();
  const monthOneYearFromNow = date.getMonth() + 12;
  return today.setMonth(monthOneYearFromNow);
};
```

Reply:
Done